modules/ROOT/pages/troubleshoot-schemas-queries.adoc

```diff
@@ -1,14 +1,182 @@
+                    "error": "Quota has been exceeded" 
+                  }
+                },
+               "classification": "DataFetchingException"
+....
+
+An exception while fetching data error occurs when the unified schema can't connect to its underlying APIs.
+
+To troubleshoot this error, ensure that:
+
+. The underlying APIs are running.
+. The URL provided for each APIs is correct. Go to *List of APIs added* > Select an API > *API details*, and confirm the URL.
+. You xref:runtime-manager::to-request-vpc-connectivity.adoc[request connection] for any of your APIs that run on standalone instances that aren’t connected to Anypoint VPC.
+. You check the `extensions` section of the error message for any details that may be useful in troubleshooting the error. 
+
+=== Other Runtime Errors
+
+* Can't prettify an invalid query 
++
+If you attempt to prettify an invalid query, DataGraph generates a syntax error. To resolve the issue, correct the syntax and then prettify the query. 
+
+* Invalid action error 
++
+If you perform an action in the DataGraph editor that creates an invalid schema, DataGraph generates an error. If you believe the action is valid, contact MuleSoft support.  
+
+== Troubleshooting Schema Errors
+
+If you encounter runtime errors when attempting to generate an API schema, troubleshoot them as follows.
+
+=== Potential Errors When Generating an API Schema
 
-** xref:schema-validation.adoc[Potential Errors When Generating an API Schema] +
 When you add a new API to your unified schema, Anypoint DataGraph generates an API schema for it. However, not all features of an API specification are successfully generated into an API schema.
-** xref:resolve-runtime-errors.adoc[Troubleshoot Runtime Errors When Running Queries] +
-If you encounter runtime errors when attempting to use the Anypoint DataGraph query editor, you must take additional steps to resolve them.
-** xref:resolve-conflicts.adoc[Resolve Conflicts and Merge Inconsistencies] +
-You must resolve any merge conflicts or merge inconsistencies when adding object types to the unified schema.
-** xref:troubleshoot-query-logs.adoc[Troubleshoot Queries With Response Logs] +
-Anypoint DataGraph provides response logs that contain useful troubleshooting information about queries.
-** xref:troubleshoot-query-traces.adoc[Troubleshoot Query Performance with Query Tracing] +
-Query tracing in Anypoint DataGraph helps you analyze query performance by tracing every call made to the source APIs of the unified schema for a given query.
+
+Anypoint DataGraph does not translate the following parts of an API specification into an API schema:
+
+* Requests with a `Media type` value other than `application/json`
+* Responses with a `Media type` value other than `application/json`
+* Response payloads with any of the unsupported types `Schema`, `File`, `Tuple`, `Matrix`, `Nil`, or arrays of any of these types
+* `query params` and `headers` with types other than scalars or scalar array
```

Comment:
The terms scalars and array are inconsistent here.

Use scalar and array or
scalars and arrays